data/hai/hai reveal 3 early.txt

```diff
@@ -194,31 +194,10 @@ mission "Hai Building Mountaintop [1b] Return"
 
 event "role-play station before construction"
 	system "Ultima Thule"
-		object
-			sprite star/f0
-			period 10
-		object
-			sprite planet/desert5
-			distance 200
-			period 22.3607
-		object
-			sprite planet/europa-b
-			distance 403.41
-			period 64.056
-		object
-			sprite planet/desert7-b
-			distance 824.9
-			period 187.302
-		object
-			sprite planet/gas1
-			distance 1706.99
-			period 557.553
-			object
-				sprite planet/ice8-b
-				distance 216
-				period 11.1964
-		object "Wormhole Alpha"
-			sprite planet/wormhole
+		remove object
+			distance 4208.63
+			period 2158.5
```

Comment:
Isn't this the first event? ~~Please move this event closer to where it is actually called, I don't know why anyone would suggest moving it here while still using it in the first file but any such suggestions were very wrong. But do that in a different PR so the changes made to this event in this one can still be reviewed easily.~~

Reply:
Good catch, I'll fix that tonight.

Reply:
Nope, nevermind, the first event actually is in HR1 - there's one mission here in HR3 that fires an event when you accept the mission *and* an event when you complete the mission.

This is why these two are together, they're used in the same mission.